lib/cmdledit.g

```diff
@@ -751,10 +751,108 @@ GAPInfo.CommandLineEditFunctions.Functions.(INT_CHAR(' ')) :=
                 GAPInfo.CommandLineEditFunctions.Functions.SpaceDeletePrompt;
 BindKeysToGAPHandler(" ");
 
+
+BindGlobal("STANDARD_EXTENDERS", rec(
+  caseSensitive := function(cand, word)
```

Comment:
What is cand, what is word, what is this function supposed to do? Again, I can of course reverse engineer it, by grepping for places this is used, but a brief comment would help a lot. Since there seem to be multiple such functions in this record, this could also be explained once in a comment before `STANDARD_EXTENDERS`